### PR TITLE
upd: Windows install command added.

### PR DESCRIPTION
**Pull request by ramazansancar**
https://github.com/gnojus/wedl/pull/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,23 +21,83 @@ Options:
...
 ```
+
+### Windows
+
+Compile from source:
+
+```cmd
+git clone https://github.com/gnojus/wedl.git
+cd wedl
+go mod download
```
Comment (gnojus) [inline README.md:57]: no need to run `go mod download`. go build will to this automatically.

File: README.md [hunk 2]
```diff
@@ -18,23 +21,83 @@ Options:
...
+
+# Build
```
Comment (gnojus) [inline README.md:59]: AFAIK cmd uses `::` as comments, so this shoud be `:: Build`.  ditto for others in this windows block

**Review by gnojus — CHANGES_REQUESTED** [review]
Thanks, see my comments

File: README.md [hunk 3]
```diff
@@ -18,23 +21,83 @@ Options:
...
+### Others
+
+Or Run the executable:
+
+```cmd
+# Help
+go run . --help
+
+# Standart Download
+go run . https://go.wetransfer.com/responsibility #Download to stdout
```
[the remark this answers is not in the document]
Comment (ramazansancar) [reply README.md:95]: I don't quite understand what you mean here. Can you give a little more detail? @gnojus

**Comment by ramazansancar** [comment]
Check pls @gnojus

**Comment by gnojus** [comment]
Hi, sorry for the long delay. I merged in your changes and did some cleanup afterwards. Thanks.

**Comment by ramazansancar** [comment]
Thanks for editing and Merge

**Pull request merged**